source/adios2/toolkit/interop/hdf5/HDF5Common.cpp

```diff
@@ -1709,6 +1709,16 @@ void HDF5Common::StaticGetAdiosStepString(std::string &stepName, size_t ts)
     stepName = "/Step" + std::to_string(ts);
 }
 
+void HDF5Common::CheckVariableOperations(const core::VariableBase &variable) const
+{
+    if (!variable.m_Operations.empty())
+    {
+        helper::Throw<std::runtime_error>("Toolkit", "interop::hdf5::HDF5Common",
+                                          "CheckVariableOperations",
+                                          "Operators not supported for HDF5 engine");
```

Comment:
Thanks for adding this warning. 
I think the error message would  be clear if it says something like "ADIOS Operators are not supported for HDF5 engine".  What would you think?

Reply:
Happy to change the wording; see forced push